src/pipecat/services/gladia.py

```diff
@@ -1,13 +1,8 @@
-#
-# Copyright (c) 2024, Daily
-#
-# SPDX-License-Identifier: BSD 2-Clause License
-#
-
 import base64
 import json
 from typing import AsyncGenerator, Optional
 
+import requests
```

Comment:
we shouldn't introduce another dependency, we should use `aiohttp`. i got rid of `requests` a while back.

Reply:
Done!